.github/workflows/reusable-generate-mermaid-svg.yaml

```diff
@@ -0,0 +1,65 @@
+#########################################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#########################################################################################
+
+name : 'generate static Mermaid files'
+on:
+  workflow_call:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: install mermaid cli
+        run: |
+          npm install -g @mermaid-js/mermaid-cli
+      - name: print mmdc version
+        run: |
+          mmdc -V
+      - name: checkout source repo
+        uses: actions/checkout@v3.5.3
```

Comment:
I wouldn't pin it to the bugfix level. On almost every other repo, we pin it to the major `@v3` version